docs/computer/student_Reports/yash_pranshi_saloni_2020.md

```diff
@@ -0,0 +1,97 @@
+---
+Title: Report for the tech stack learned till now at AUV club, ZHCET
```

Comment:
You have made them go uppercase. Write the same way as Jawad suggested.

Reply:
Make it title, author and email-id.

Reply:
ok